<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slidesForm</class>
 <widget class="QWidget" name="slidesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="slides_next">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>385</y>
     <width>155</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="type" stdset="0">
    <string>typeB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slides_back">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>385</y>
     <width>155</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="type" stdset="0">
    <string>typeB</string>
   </property>
  </widget>
  <widget class="QLabel" name="slides_slide">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>326</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>slides_slide</zorder>
  <zorder>slides_back</zorder>
  <zorder>slides_next</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
